src/napari_spatialdata/_viewer.py

```diff
@@ -539,12 +548,13 @@ def add_sdata_points(self, sdata: SpatialData, key: str, selected_cs: str, multi
         np.fliplr(xy)
         # radii_size = _calc_default_radii(self.viewer, sdata, selected_cs)
         radii_size = 3
+        version = get_napari_version()
+        kwargs = {"edge_width": 0.0} if version <= packaging.version.parse("0.4.19") else {"border_width": 0.0}
```

Comment:
I would say this solution is very temporary. 0.4.19 had some fixes with layer.events.data that will be important for the table widget so we would need to have 0.4.19 as constraint.